packages/broker/src/config/ConfigWizard.ts

```diff
@@ -153,7 +156,41 @@ const createPluginPrompts = (): Array<inquirer.Question | inquirer.ListQuestion
         default: true
     }
 
-    return [selectApiPluginsPrompt, ...portPrompts, minerPluginPrompt]
+    const wantToSetBeneficiaryAddressPrompt: inquirer.DistinctQuestion = {
+        type: 'confirm',
+        name: 'wantToSetBeneficiaryAddress',
+        // eslint-disable-next-line max-len
+        message: `By default the generated / imported Ethereum identity will be used for staking, would you like to change this to a different address (i.e. set a beneficiary address)?`,
+        default: false,
```

Comment:
Not sure if it is possible to make this prompt less complex... Is the core question something like this:
```
You can mine the tokens to your own wallet (the imported Ethereum identity provided at step X) or some other wallet. That wallet should contain the stake.`
```
Could also consider using enum instead of boolean?
```
Choose wallet: own / other
```

Reply:
"Ethereum identity" could cause some confusion ("Ethereum address" ditto), considering far from everyone understands that Ethereum and Polygon addresses are one and the same. Change to "Polygon wallet address"?

Reply:
> Change to "Polygon wallet address"?

Hmm, yeah good point. This is probably easier to understand.